plugins/Morpheus/stylesheets/general/_forms.less

```diff
@@ -72,23 +72,50 @@ input.browser-default {
   background-color: @theme-color-brand !important;
 }
 
-[type="checkbox"]:checked+label:before {
+[type="checkbox"]:checked + label::before {
   border-right-color: @theme-color-brand;
   border-bottom-color: @theme-color-brand;
 }
+
 nav.blue-grey.darken-3 {
   background-color: @theme-color-header-background !important;
 }
-[type="radio"]:checked+label:after, [type="radio"].with-gap:checked+label:after {
+
+[type="radio"]:checked + label::after,
```

Comment:
same here where it adds a colon

Reply:
The browser already auto corrects that to `::after`.

![Screenshot_20190411_085346](https://user-images.githubusercontent.com/22813377/55936702-545f5000-5c37-11e9-8f59-78120e37a2f9.png)

Reply:
This is after the changes, as you see the code that gets used is the same, only the line numbers changed.

![Screenshot_20190411_093428](https://user-images.githubusercontent.com/22813377/55939074-20872900-5c3d-11e9-9112-9fa67aee26ce.png)